bloxid/hashids.go

```diff
@@ -0,0 +1,102 @@
+package bloxid
+
+import (
+	"errors"
+
+	hashids "github.com/speps/go-hashids/v2"
+)
+
+const (
+	hashIDAllowedChar = "0123456789abcdef"
+	maxHashIDLen      = 36
+	hashIDPrefix      = "HIDZ"
```

Comment:
please add comment that the prefix needs to be uppercase so there are no collisions with chars in hashIDAllowedChar